_includes/markdown/React.md

```diff
@@ -0,0 +1,130 @@
+* Is performance a primary concern? Will your project need to render large amounts of data in real time while remaining performant for the browser?
+* Are you building a native mobile app for iOS and/or Android? The React team also maintains React Native, a subset of React which makes it possible to write applications in JavaScript using React components that compile to native Swift and Java.
+
+React is easily integrated into specific parts of the front-end or admin of an existing site, but it can also be used to render entire sites, effectively replacing traditional WordPress templates—although doing so requires a lot more planning and scaffolding of features (such as routing) that would normally be handled by the CMS out of the box.
+
+## Implementing React on a Project
+
+There are a number of ways to get up and running with React in a project. The deciding factor is based on the context of your project and how you plan to implement React. The following subsections will outline instructions that are specific to installing React on a per project basis:
+
+### Script Element
+
+Including React via a script element in your HTML is a pretty quick and standard way of including the library. To do this you can either [download the library directly](https://react-cn.github.io/react/downloads.html), use the [CDN](https://reactjs.org/docs/cdn-links.html), or install through [NPM](https://www.npmjs.com/package/react) and direct-link into your `node_modules` directory.
+
+Something to note about this method is that React and ReactDOM are both required. There is also no inherent support for ES6 or JSX without Webpack or Babel in this context. You would need to separately configure a transpiler or write in Vanilla JavaScript.
+
+### Including with NPM
+
+If you already have a Node / NPM / Yarn project that you wish to include React as a dependency of, then you can use the installation instructions found on the [Official React Documentation](https://reactjs.org/docs/add-react-to-an-existing-app.html). Include React and ReactDOM in your package file just like you would any other dependency. Also be sure your transpiler is working to correctly process the ES6 and JSX from React into something the browser can understand.
+
+### Installation with a CLI
+
+[The React CLI](https://reactjs.org/blog/2016/07/22/create-apps-with-no-configuration.html) (create-react-app) provides us a quick way to setup and scaffold a React project. It makes use of the modern frontend workflows and out of the box, React CLI provides hot reloading, minification, asset management, module bundling, linting and a development server to test your application under realistic circumstances. It works with zero configuration from your part as everything is already set up.
+
+Using the CLI tool is ideal if you are creating a full React App with an API backend.
```

Comment:
Would this be integrated into a WordPress theme?

Reply:
Doubtful, most likely just consuming the API like we did on Uber

Reply:
Could be used for a plugin though